sources/packages/web/src/views/institution/LocationApplications.vue

```diff
@@ -98,6 +98,15 @@ export default {
     const goToViewApplication = (applicationId: number) => {
       console.log(applicationId);
     };
+    watch(
+      () => props.locationId,
+      async currValue => {
```

Comment:
Can you please create one method and call it on watch and onMounted?
The method is supposed to be called in the same way for both, right?

Reply:
okay, I didn't create a common function because there was only one line of operation. anyways, I have updated the code.